lib/main.coffee

```diff
@@ -79,6 +85,34 @@ Notifications =
 
     @isInitialized = true
 
+  togglePanel: ->
```

Comment:
@UziTech it looks like this commit reintroduced code that was removed in https://github.com/atom/notifications/pull/177.

...which makes me slightly worried that this may have happened in other places as well.

Reply:
Ya sorry, that must have been reintroduced on a rebase. I did a quick glance through the code and it looks like that is the only code that was reintroduced.